service/resource/legacyv2/service_account.go

```diff
@@ -0,0 +1,28 @@
+package legacyv2
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	api "k8s.io/api/core/v1"
+	apismeta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func newServiceAccount(customObject v1alpha1.FlannelConfig, name string) *api.ServiceAccount {
+	app := networkApp
```

Comment:
Could we use `networkApp` instead of defining a new variable?

Reply:
Agree. I took it from other resource but I don't find a good reason